typings/resin-sdk.d.ts

```diff
@@ -179,13 +213,53 @@ declare namespace ResinSdk {
 		twoFactorRequired?: boolean;
 		username: string;
 
+		owns__organization: NavigationResource<Organization>;
+
+		user__is_member_of__organization: ReverseNavigationResource<Organization>;
+		user__is_member_of__team: ReverseNavigationResource<Team>;
 		creates__release: ReverseNavigationResource<Release>;
 		owns__device: ReverseNavigationResource<Device>;
-		user__is_member_of__application: ReverseNavigationResource<Application>;
 		// this is what the api route returns
 		social_service_account: ReverseNavigationResource<SocialServiceAccount>;
 	}
 
+	type OrganizationMembershipRoles = 'owner' | 'administrator' | 'member';
+
+	interface OrganizationMembershipRole {
+		id: number;
+		name: OrganizationMembershipRoles;
+	}
+
+	interface OrganizationMember {
+		id: number;
+		created_at: string;
+
+		grants__organization_membership_role: NavigationResource<
+			OrganizationMembershipRole
+		>;
+		is_member_of__organization: NavigationResource<Organization>;
+		user: NavigationResource<User>;
+	}
+
+	type ApplicationMembershipRoles =
+		| 'collaborator'
```

Comment:
Ditto - I think we're planning to remove the explicit 'collaborator' role entirely once orgs is merged, right?

Reply:
I'm not sure about this, it's in the Orgs API PR and does whatever developers can do + managing the team memberships (like a team "admin").
See: https://github.com/resin-io/resin-api/pull/1079/files#diff-e4921799df81f37704fbab97e47687e3R360

Reply:
Based on https://www.flowdock.com/app/rulemotion/p-billing/threads/qBB-ePfPceQRwimkrCzy99pqWQ9 I don't think that's true:

> Why does that mean we still need a collaborator role in here?
> 
> We don't, this will be removed in Orgs, and  replaced with owner

I think that means this should be `owner`. We should check with @afitzek though.